allowlist.json

```diff
@@ -58,6 +58,12 @@
             "display_name": "Air theme for landing pages",
             "minimum_mautic_version": "4.3.0",
             "maximum_mautic_version": null
+        },
+        {
+            "package": "matbcvo/mautic-amazon-sns-callback",
+            "display_name": "Integrates with Amazon SNS (Simple Notification Service) to handle bounce, complaint, and other notifications for email deliverability.",
+            "minimum_mautic_version": "5.0,0",
```

Comment:
Oops. There is a typo. A comma instead of the dot

Reply:
Martin noticed it. He'll create a PR for this typo fix.

Reply:
Created PR #16